_data/en/settings.yml

```diff
@@ -316,24 +316,6 @@ policies:
           url: '#terms'
     - name: Rules of use
       url: '/policy/rules-of-use/'
-who-uses-login-page:
-  hero:
-    heading: Who uses login.gov?
-    text: Login.gov is a trusted, government-issued sign in service.  The public uses login.gov for simple and secure access to the services our government partners provide.
-  partners: |-
-    ## Our partners
-
-    Our participating government partners offer a variety of services such as:
-
-    * USAJOBS (Office of Personnel Management)
-    * Paycheck Protection Program (Small Business Administration)
-    * Disaster Loan Applications Program (Small Business Administration)
-
-    <div class="mobile partners-img" aria-hidden="true"></div>
```

Comment:
It looks like we lost this image in the conversion:

Before|After
---|---
![image](https://user-images.githubusercontent.com/1779930/121385435-1b09f400-c917-11eb-8a22-3f183cfbfede.png)|![image](https://user-images.githubusercontent.com/1779930/121385456-20673e80-c917-11eb-9c7a-308524654fd7.png)

I might also be concerned to have this as part of the Netlify content, since it could be easy to mistakenly remove, depending how well their WYSIWYG editor handles arbitrary HTML.

But because it's the same for French and Spanish, maybe we can just add it to `_includes/components/who-uses-login.html` at the end of `<div class="partners list">`?